pwndbg/gdblib/disasm/arm.py

```diff
@@ -91,6 +91,8 @@ def _set_annotation_string(self, instruction: PwndbgInstruction, emu: Emulator)
     @override
     def _condition(self, instruction: PwndbgInstruction, emu: Emulator) -> InstructionCondition:
         if instruction.cs_insn.cc == ARM_CC_AL:
+            if instruction.id in (ARM_INS_B, ARM_INS_BL, ARM_INS_BLX, ARM_INS_BX, ARM_INS_BXJ):
+                instruction.declare_conditional = False
```

Comment:
Like in AArch64, these Arm instructions can be conditional based on the condition code. This change reflects this fact in the disasm view (checkmarks are rationed out correctly, and jumps are unrolled correctly now)